### PR TITLE
Ends the confusing inconsistency of power cable and CONDUCT weapons

### PR DESCRIPTION
**Pull request by Robustin**
No meme lives forever my son. 

This mechanic was horribly unintuitive and didn't enhance the game in any way.

Even IF you get away from the absurdity of getting shocked by insulated power cables on contact, the CONDUCT flag used to determine if you get shocked was horribly inconsistent:
- Lit welder shocks you
- Telescopic baton doesn't
- Metal rod shocks you
- Metal chair doesn't
- Wooden Staff of Doors shocks you
- Chainsaws do not

It's just a bad system all around, if you still want to watch innocent people get shocked to death then just hang around an exposed grille.

### CODE REVIEW

**Comment by ChangelingRain** [comment]
A better fix would be to add or remove the conduct flag from the offending items, rather than remove the mechanic because some items have or don't have the flag when they should/shouldn't.

File: code/modules/power/cable.dm [hunk 1]
```diff
@@ -28,7 +28,7 @@ By design, d1 is the smallest direction and d2 is the highest
 	on_blueprints = TRUE
 	var/datum/powernet/powernet
 	name = "power cable"
-	desc = "A flexible, superconducting cable for heavy-duty power transfer."
+	desc = "A flexible, superconducting insulated cable for heavy-duty power transfer."
```
Comment (ChangelingRain) [inline code/modules/power/cable.dm]: If you have to add it to the desc, clearly it isn't insulated.

**Comment by MrPerson** [comment]
Shocking people from an errant click sounds extremely unfun

**Comment by MMMiracles** [comment]
The amount of times I've died because of accidently clicking a thin red line while fighting in maintenance is more than I'd like to admit.

File: code/modules/power/cable.dm (hunk 1 above)
Comment (Robustin) [inline code/modules/power/cable.dm]: The desc change was half-meme for the people who were claiming that Nanotrasen just paints live cables.

**Comment by Robustin** [comment]
Yea, this has been on my to-do list for a while. Its especially bad when fighting on the "vertical" sections of maint, where basically every click was a decent chance to shock yourself.

**Comment by duncathan** [comment]
:+1:

**Comment by KazeEspada** [comment]
Conduct is still used when sticking items in light sockets.

**Comment by Dorsisdwarf** [comment]
:-1: 
Cables are totally not insulated. Deathtrap, remember?

**Comment by Robustin** [comment]
The net exchange on the "Deathtrap, remember?" meme is still more than positive when you have tesla-grilles. There's good aspects of being on a deathtrap, but having a pixel-wide strip of death all over maint is not one of them.

**Comment by Dorsisdwarf** [comment]
Ah, I see now

**Comment by Iamgoofball** [comment]
As long as this still shocks people trying to cut the wires uninsulated then ok

**Comment by Lumipharon** [comment]
Doesn't this make laying cables as a pleb much safer?
Like, you can already lay live cables without gloves if you're not dumb, but most people aren't, so can't.

**Comment by Robustin** [comment]
This doesn't affect cutting cables.

Cutting cables is a guaranteed shock without gloves. Laying cables? I'd need to see if the shocking solely came from hitting cable with cable instead of some other section of code. If it was, then yea laying cable would get you shocked a lot less, if that's an issue for sadists out there it wouldn't be hard to another section like wirecutters that would punish you for laying live cable w/o gloves.

**Comment by Lumipharon** [comment]
Currently, you can safely extend live cables, simply by standing and clicking correctly.
If you try extending the cable by instead clicking on the pre-existing cable, it shocks you. 

It's a small thing, but it helps seperate the plebs from the expert autism fort creators.

in general I'm against this change, because making things needlessly safer is gay, but fighting in maint and accidentally clicking a wire IS super shit and makes little sense - you're running over the cables just fine, yet you try swing a punch at a guy in the face but miss and somehow hit the cable on the floor and you just die.

**Comment by Shapsy** [comment]
Then why would you be against this change if that's what this gets rid of?

**Comment by Robustin** [comment]
Because they're more focused on the laying-live-cable-with-bare-hands aspect.

It's a bit silly Lumi because:

1) The impact that has on the game is far less than the daily deaths from people getting ruined in combat with wire clicks. 

2) You acknowledged yourself that a skilled player knows that you can avoid shocks by where you want the wire to appear and not where the wire will be sourced from. So this just makes it easier to lay live wire without collecting some eclectic obscure info on wire-laying.

I mean how many players are good enough at Engineering to have some project that requires laying down new wire, but isn't good enough to know the trick to avoid getting shocked? 

If we really wanted to not make the game mindlessly safer we should make ALL live wire-laying shock anyone who's not insulated, but then there'd be a new host of complaints about that.

**Comment by Lumipharon** [comment]
Because it makes shit like snipping all the wires in maint easier since you no longer need gloves to grief the fuck out of engineering, and having things be doable with the proper knowledge, while fucking you in the ass if you lack said knwledge is good.

But yes, I appreciate that clicking on wire mid fight is shitty, and that the change might be ultimately needed despite my other points.

**Comment by Iamgoofball** [comment]
Wait what lumi

This doesn't prevent uninsulated wirecutting

**Comment by Lumipharon** [comment]
Oh, just for hitting it with random shit then? yeah whatevs, I ain't no codeman.

**Comment by Razharas** [comment]
I approve

**Pull request merged**